Misc/NEWS.d/next/Build/2019-04-29-09-57-20.bpo-36747.1YEyu-.rst

```diff
@@ -0,0 +1 @@
+Remove the stale scriptsinstall Makefile target
```

Comment:
nitpick: would you mind to add a "." at end of your sentence? :-)

Reply:
Done.